Alexey-budgetCalc/main.py

```diff
@@ -0,0 +1,53 @@
+import modul
+
+"""1.Add new payment
+To add new information
+2.Delete payment
+To delete payment by index
+3.Show all
+To show all payments witch indexes
+4.Sort by category
+To sort by one category
+5.Sort by categories
+To sort by 1 or more categories
+6.Sort by month
+To sort by one month
+7.Sort by few
+To sort by month and categories
+0.Exit
+To close the program"""
+
+
+def menu():
+    data = modul.information(budget_file="budget.csv")
```

Comment:
Its better to separate strings to constants at the top of module.

`STORAGE_FILENAME="budget.csv"`

And use it further to perform bulk update of all names across the app